.github/workflows/nightly.yml

```diff
@@ -69,15 +49,7 @@ jobs:
     - name: Swap in nightly c2pa-rs build
       run: |
         sed -i "s/^c2pa = { version = \"[^\"]*\",/c2pa = { git = \"https:\\/\\/github.com\\/contentauth\\/c2pa-rs.git\", branch = \"nightly\",/;" Cargo.toml
-  
```

Comment:
are we no longer generating a changelog, or did that move somewhere else? It seems like change logs would be particularly useful on nightly builds, since people will be experimenting with new features.

Reply:
I removed it for consistency with the c2pa-rs process, but I'm open to adding it back.

Reply:
Changelogs are now preserved